support/client/lib/mil-sym/cws.js

```diff
@@ -5599,104 +5848,205 @@ define( function(){
                   "symbolID": "G*FPACSI--****X",
                   "tag": "FIRE SUPPORT AREA IRREGULAR",
                   "symbolType": "MultiSegmentPolygon",
                   "validModifiers": [ "uniqueDesignation1", "DTG1", "DTG2" ]
             },
+            "TACGRP.FSUPP.ARS.C2ARS.ACA.RTG": {
+                  "symbolID": "G*FPACAI--****X",
+                  "tag": "AIRSPACE COORDINATION AREA RECTANGULAR",
+                  "symbolType": "RectangleMultiPoint",
+                  "validModifiers": [ "uniqueDesignation1", "additionalInfo1", "additionalInfo2", "additionalInfo3", "DTG1", "DTG2" ]
+            },
             "TACGRP.FSUPP.ARS.C2ARS.ACA.IRR": {
                   "symbolID": "G*FPACAI--****X",
                   "tag": "AIRSPACE COORDINATION AREA IRREGULAR",
                   "symbolType": "MultiSegmentPolygon",
                   "validModifiers": [ "uniqueDesignation1", "additionalInfo1", "additionalInfo2", "additionalInfo3", "DTG1", "DTG2" ]
             },
+            "TACGRP.FSUPP.ARS.C2ARS.FFA.RTG": {
+                  "symbolID": "G*FPACFI--****X",
+                  "tag": "FREE FIRE AREA RECTANGULAR",
+                  "symbolType": "RectangleMultiPoint",
+                  "validModifiers": [ "uniqueDesignation1", "DTG1", "DTG2" ]
+            },
             "TACGRP.FSUPP.ARS.C2ARS.FFA.IRR": {
                   "symbolID": "G*FPACFI--****X",
                   "tag": "FREE FIRE AREA IRREGULAR",
                   "symbolType": "MultiSegmentPolygon",
                   "validModifiers": [ "uniqueDesignation1", "DTG1", "DTG2" ]
             },
+            "TACGRP.FSUPP.ARS.C2ARS.NFA.RTG": {
+                  "symbolID": "G*FPACNI--****X",
+                  "tag": "NO-FIRE AREA RECTANGULAR",
+                  "symbolType": "RectangleMultiPoint",
+                  "validModifiers": [ "uniqueDesignation1", "DTG1", "DTG2" ]
+            },
             "TACGRP.FSUPP.ARS.C2ARS.NFA.IRR": {
                   "symbolID": "G*FPACNI--****X",
                   "tag": "NO-FIRE AREA IRREGULAR",
                   "symbolType": "MultiSegmentPolygon",
                   "validModifiers": [ "uniqueDesignation1", "DTG1", "DTG2" ]
             },
+            "TACGRP.FSUPP.ARS.C2ARS.RFA.RTG": {
+                  "symbolID": "G*FPACRI--****X",
+                  "tag": "RESTRICTIVE AREA RECTANGULAR",
+                  "symbolType": "RectangleMultiPoint",
+                  "validModifiers": [ "uniqueDesignation1", "DTG1", "DTG2" ]
+            },
             "TACGRP.FSUPP.ARS.C2ARS.RFA.IRR": {
                   "symbolID": "G*FPACRI--****X",
                   "tag": "RESTRICTIVE AREA IRREGULAR",
                   "symbolType": "MultiSegmentPolygon",
                   "validModifiers": [ "uniqueDesignation1", "DTG1", "DTG2" ]
             },
+            "TACGRP.FSUPP.ARS.C2ARS.SNSZ.RTG": {
+                  "symbolID": "G*FPACEI--****X",
+                  "tag": "SENSOR ZONE RECTANGULAR",
```

Comment:
I don't see this one in the list of options.

Reply:
That was a copy/paste error.  Those had the same named entries as the ones below them (dead space areas), so the sensor zones were being redefined.